client.go

```diff
@@ -31,6 +34,31 @@ type authResponse struct {
 	TokenType   string `json:"token_type,omitempty"`
 }
 
+type organizationsResponse struct {
```

Comment:
These structs that are representations of responses from the HCP Vault Secrets API should not be in the `client.go` file. I'd expect to see them in the `secret.go` file instead or their own `organizations.go` file with corresponding tests.